In Dart you can capture any exception object that you caught:

```dart
import 'package:sentry/sentry.dart';

void main() async {
  try {
    aMethodThatMightFail();
  } catch (exception, stackTrace) {
    await Sentry.captureException(
      exception,
      stackTrace: stackTrace,
    );
  }
}
```
